### PR TITLE
Remove serverspec deps from all serverspecs, and ensure directory exists

### PR DESCRIPTION
**Pull request by davidcusatis**

### CODE REVIEW

**Comment by pwplusnick** [comment]
LGTM

File: roles/ceilometer-common/tasks/serverspec.yml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 ---
+- name: ensure serverspec directory exists
+  file: dest=/etc/serverspec/spec/localhost state=directory
+            owner= root mode=0755 recurse=true
```
Comment (j2sol) [inline roles/ceilometer-common/tasks/serverspec.yml]: You've got a space issue, between owner= and root. In every one of these.

**Comment by j2sol** [comment]
ok to test

**Comment by j2sol** [comment]
Ah, the `serverspec` part is missing. I'd say toss that in defaults-2.0.yml, or you'll have to change the conditional to be more like `when: serverspec|default({'enabled': False}).enabled|bool`

**Comment by j2sol** [comment]
sorry, this branch now has conflicts and you'll have to rebase, but the good news is that tests pass...

**Pull request merged**